test/functional/http/get/accounts.js

```diff
@@ -2,319 +2,268 @@
+				return accountsEndpoint.makeRequest({publicKey: 'invalidPublicKey'}, 400).then(function (res) {
+					expectSwaggerParamError(res, 'publicKey');
 				});
 			});
 
 			it('using invalid publicKey (integer) should fail', function () {
-				return getAccountsPromise('publicKey=' + '123').then(function (res) {
-					node.expect(res).to.have.property('status').to.equal(400);
-					node.expect(res).to.have.nested.property('body.message').to.contain('Expected type string but found type integer');
+				return accountsEndpoint.makeRequest({publicKey: '123'}, 400).then(function (res) {
+					expectSwaggerParamError(res, 'publicKey');
 				});
 			});
 
 			it('using empty publicKey should return empty results', function () {
-				return getAccountsPromise('publicKey=').then(function (res) {
-					node.expect(res).to.have.property('status').to.equal(200);
-					node.expect(res).to.have.nested.property('body.accounts').that.is.an('array').to.have.length(0);
+				return accountsEndpoint.makeRequest({publicKey: ''}, 200).then(function (res) {
+					res.body.data.should.have.length(0);
 				});
 			});
 
 			it('using empty publicKey and address should fail', function () {
-				return getAccountsPromise('publicKey=&address=').then(function (res) {
-					node.expect(res).to.have.property('status').to.equal(400);
-					node.expect(res).to.have.nested.property('body.message').to.contain('String is too short (0 chars), minimum 1');
+				return accountsEndpoint.makeRequest({publicKey: '', address: ''}, 400).then(function (res) {
+					expectSwaggerParamError(res, 'address');
 				});
 			});
 
 			it('using known address and matching publicKey should be ok', function () {
-				return getAccountsPromise('address=' + node.gAccount.address + '&publicKey=' + node.gAccount.publicKey).then(function (res) {
-					validateAccountFields(res, node.gAccount);
+				return accountsEndpoint.makeRequest({publicKey: node.gAccount.publicKey, address: node.gAccount.address}, 200).then(function (res) {
+					res.body.data.should.have.length(1);
+					res.body.data[0].address.should.be.eql(node.gAccount.address);
+					res.body.data[0].publicKey.should.be.eql(node.gAccount.publicKey);
 				});
 			});
 
 			it('using known address and not matching publicKey should return empty result', function () {
-				return getAccountsPromise('address=' + node.gAccount.address + '&publicKey=' + account.publicKey).then(function (res) {
-					node.expect(res).to.have.property('status').to.equal(200);
-					node.expect(res).to.have.nested.property('body.accounts').that.is.an('array').to.have.length(0);
+				return accountsEndpoint.makeRequest({publicKey: account.publicKey, address: node.gAccount.address}, 200).then(function (res) {
+					res.body.data.should.have.length(0);
+				});
+			});
+		});
+
+		describe.only('secondPublicKey', function () {
+
+			var spkAcccount = node.randomAccount();
```

Comment:
i dont like the way this variable is shortened. Can you expand it to be the full name for readability?

Reply:
Done.